client/extensions/woocommerce/state/data-layer/ui/shipping-zones/index.js

```diff
@@ -246,14 +249,34 @@ const getZoneMethodUpdateSteps = ( siteId, zoneId, method, state ) => {
 						siteId,
 						getZoneId( zoneId, actionList ),
 						getMethodId( id, actionList ),
-						methodProps,
+						methodPropsToUpdate,
 						actionListStepSuccess( actionList ),
 						actionListStepFailure( actionList )
 					)
 				);
 			},
-		},
-	];
+		} );
+	}
+
+	if ( isWcsMethod && ! isEmpty( extraMethodProps ) ) {
+		steps.push( {
+			description: translate( 'Updating Shipping Method (extra settings)' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Updating Shipping Method' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).